### ISSUE TITLE
Type error

### ISSUE THREAD

**Issue by runnez** [RobinTail/express-zod-api#1092]
Hi there, could you please advise what's wrong? Trying to handle error 
```
Exported variable 'helloWorldEndpoint' has or is using name 'BearerSecurity' from external module "./my-app/node_modules/express-zod-api/dist/index" but cannot be named
```

```
import { defaultEndpointsFactory } from "express-zod-api";

import { z } from "zod";

export const helloWorldEndpoint = defaultEndpointsFactory.build({
  method: "get",
  input: z.object({
    // for empty input use z.object({})
    name: z.string().optional(),
  }),
  output: z.object({
    greetings: z.string(),
  }),
  handler: async ({ input: { name }, options, logger }) => {
    logger.debug("Options:", options); // middlewares provide options
    return { greetings: `Hello, ${name || "World"}. Happy coding!` };
  },
});
```

**Comment by RobinTail** [comment]
did you find a way to resolve this issue, @runnez ?

**Comment by RobinTail** [comment]
If it's not, please let me know, since I remember I fixed similar issue on "can not be named":
https://github.com/RobinTail/express-zod-api/discussions/951

**Comment by runnez** [comment]
Hey @RobinTail. Thank you for getting back to me. The issue was about using the package in monorepo. I didn't manage to handle this 😢

**Comment by RobinTail** [comment]
@runnez , I found the cause and it's duplicate of #1182 
It's fixed today by #1189 and the fix will be released today as v12.1.0, @runnez 

Please try that new version. If the issue remains, please open a new issue, @runnez
